packages/next/client/image.tsx

```diff
@@ -680,7 +714,46 @@ export default function Image({
     handleLoading(imgRef, srcString, layout, placeholder, onLoadingCompleteRef)
   }, [srcString, layout, placeholder, isVisible])
 
-  return (
+  return isRaw ? (
+    <>
```

Comment:
Is this copy/pasta from below but just changing `<span>` to `<>`?

If so, perhaps we could try to reuse the code with a function or component to avoid duplicate code

Reply:
Good call. The `raw` version of the image element was originally more different from the non-raw version, but the two have converged quite a bit. I've factored out the <img> and <noscript> elements into a component.